content/applications/sales/sales/send_quotations/deadline.rst

```diff
@@ -1,44 +1,57 @@
-sales order.
+Expiration date deadlines
+=========================
 
-.. image:: deadline/quotation_deadlines_1.png
+With Odoo *Sales*, it's possible to instantly add an expiration date in the :guilabel:`Expiration`
+field of a quotation or sales order.
+
+.. image:: deadline/quotation-deadlines-1.png
    :align: center
-   :class: img-thumbnail
-   :alt: How to configure deadlines on Odoo Sales?
+   :alt: How to configure deadlines on Odoo Sales.
+
+Deadlines in quotation templates
+================================
+
+The Odoo *Sales* application also makes it possible to add a deadline to every quotation template,
+as well. Whenever a specific quotation template is used in a quote, its associated deadline is
+automatically applied.
+
+To add a deadline to a quotation template, navigate to :menuselection:`Sales app --> Configuration
+--> Quotation Templates`, and either select the desired quotation template to which a deadline
+should be added, or click :guilabel:`Create` to build a new quotation template from scratch.
+
+Once on the quotation template detail page, clik the :guilabel:`Edit` button to edit the quotation.
 
-Use deadline in your quotation templates
-========================================
+Then, add a specific number of days to the :guilabel:`Quotation expires after` field, located
+beneath the quoation template name. The number of days represents how long the quotation will be
+valid for, before it expires. When done, click :guilabel:`Save`.
 
-It is also possible to add a deadline to every quotation template created. Whenever a specific
-quotation template is used in a quote, its associated deadline will be automatically applied. Be
-sure to check out our documentation about :doc:`quote_template` to excel in their use.
+Be sure to check out the documentation related to :doc:`quote_template` to learn more.
 
-.. image:: deadline/quotation_deadlines_2.png
+.. image:: deadline/quotation-deadlines-2.png
    :align: center
-   :class: img-thumbnail
-   :alt: How to use deadline in a quotation template on Odoo Sales?
+   :alt: How to use deadline in a quotation template on Odoo Sales.
 
 .. tip::
-   By clicking on the **Customer Preview** button, you will be able to see when the offer expires.
-   For your information, the number of days will be the same as those mentioned in the quotation
-   template.
+   By clicking on the :guilabel:`Customer Preview` button on a quotation, Odoo clearly displays
+   when that specific offer expires. As a reminder, the number of days is the same as those
+   mentioned in the quotation template (if a quotation template was used for the initial
+   quotation).
 
-   .. image:: deadline/quotation_deadlines_3.png
+   .. image:: deadline/quotation-deadlines-3.png
       :align: center
-      :class: img-thumbnail
-      :alt: How customers will see deadlines on Odoo Sales?
+      :alt: How customers will see deadlines on Odoo Sales.
 
 .. seealso::
    - :doc:`quote_template`
```

Comment:
No need to use a bullet point in a single seealso, and please use a global relative path rather than a local relative path